src/Java.Interop.Tools.Cecil/Java.Interop.Tools.Cecil/TypeDefinitionRocks.cs

```diff
@@ -6,32 +6,30 @@
 namespace Java.Interop.Tools.Cecil {
 
 	public static class TypeDefinitionRocks {
+		static TypeDefinition ResolveCached (this TypeReference type, IMetadataResolver? resolver) =>
```

Comment:
Not sure why this method is needed *at all* given that we're now requiring the use of `IMetadataResolver` everywhere.